sync/sync_head.go

```diff
@@ -238,7 +243,7 @@ func (s *Syncer[H]) verifyBifurcating(ctx context.Context, subjHead, networkHead
 	}
 
 	s.metrics.failedBifurcation(ctx, int64(networkHead.Height()), networkHead.Hash().String())
-	log.Warnw("sync: header validation against subjHead", "height", networkHead.Height(), "hash", networkHead.Hash().String())
+	log.Warnw("header validation against subjHead", "height", networkHead.Height(), "hash", networkHead.Hash().String())
```

Comment:
IMO should be error log and say header validation failed against subjHead

Reply:
Conflict-ish with a previous review comment https://github.com/celestiaorg/go-header/pull/219#discussion_r1871617234

(also, you have commented on the old commit, so your comment might be no-op by now).

Reply:
I am ok with it being an error log, but I don't think it should be a header validation error. It's a bifurcation failure as a follow up to previous warn about bifurcation being started